### DISCUSSION TITLE
Tagging labels when saving in Android?

### DISCUSSION THREAD

**Discussion by hiepph** [omnivore-app/omnivore#2986] [Ideas]
In Android, when I share/save an article to Omnivore, it shows just a plain notification "Saved to Omnivore". I couldn't tag any label.

It would be cool to integrate the tagging/labelling feature that's available like the web extension.

**Comment by johnlunney** [comment]
(I also opened an [Issue](https://github.com/omnivore-app/omnivore/issues/3167) for Android share with selected text, to make that a highlight)
